ai21/clients/bedrock/resources/bedrock_completion.py

```diff
@@ -46,6 +46,47 @@ def create(
         if model_id is None:
             raise ValueError("model_id should be provided in either the constructor or the 'create' method call")
 
-        raw_response = self._invoke(model_id=model_id, body=body)
+        raw_response = self._post(body=body, model_id=model_id)
```

Comment:
Is there a reason to keep this class? Can we delete this and use `ai21/clients/studio/resources/chat/chat_completions.py` instead?